connector/mqtt/vernemq/v2k-bridge/app/app.js

```diff
@@ -1,13 +1,19 @@
 const { Logger } = require('@dojot/microservice-sdk');
 
-const { app: appConfig } = require('./config');
+const util = require('util');
+const { unflatten } = require('flat');
+
+const config = require('./config');
 const AgentMessenger = require('./AgentMessenger');
 
 // Logger configuration
-Logger.setTransport('console', appConfig.logger.transports.console);
-Logger.setVerbose(appConfig.logger.verbose);
+const logConfig = unflatten(config.logger);
```

Comment:
In the future, we could handle this in the sdk.

Reply:
Yes, I've planned to treat this when I'm done with the ConfigManager.